### ISSUE TITLE
MULT DX, DY, DZ values do not match MESH XB values.

### ISSUE THREAD

**Issue by marcosvanella** [firetools/qgis2fds#91]
Ciao Ema, this is from an issue in fds #13479 (https://github.com/firemodels/fds/issues/13479) . The FDS mesh written for the case in the issue is:
```

&MULT ID='Meshes'
      DX=346.59 I_LOWER=0 I_UPPER=4
      DY=278.26 J_LOWER=0 J_UPPER=7 /
&MESH IJK=34,27,33 MULT_ID='Meshes'
      XB=-866.59,-520.01,-1113.01,-834.76,211.57,543.35 /

```

The values of DX and DY do not match exactly the XMAX-XMIN, YMAX-YMIN being written (they differ by the last digit). Turns out this difference make distance among meshes to increase as you set more meshes in X,Y. This is leading to issues matching meshes. 
Can we make sure these numbers match when written?
Thanks!
M

**Comment by ericvmueller** [comment]
Is this the same as here https://github.com/firetools/qgis2fds/issues/57? Did that fix ever make it into the main branch?

**Comment by johodges** [comment]
My understanding is the main branch is still behind the dev branch. That fix is in place on dev but the user may be using main.

**Comment by emanuelegissi** [comment]
Yes, the last release is behind the dev branch, this is why the fix didn't get to the users.
I will to sync that asap.

**Comment by 3dfirelab** [comment]
I was using version 1.0.2
from the qgisinstallation plugin.
So no need to share with you my qgis project?

**Comment by emanuelegissi** [comment]
No need to share the project. The issue is known and fixed in the dev version. I shall only release a newer version.
